### DISCUSSION TITLE
How to show separator lines in between the columns?

### DISCUSSION THREAD

**Discussion by g-sharat** [Live-Charts/LiveCharts2#641] [Q&A]
I want to show a seperator line in between the columns rather than on column. Is there a way I can achieve it? (Shown in purple line in the image) Thanks

![Screenshot_1663930432](https://user-images.githubusercontent.com/70692056/191964393-66a25a75-8a01-4d38-b6ab-5b1ee6bbc281.png)

**Comment by beto-rodriguez** [comment]
I moved this to an issue in #810, this will be supported in the next version of the library.
